pkg/certificate/store/nssdb_unix.go

```diff
@@ -0,0 +1,125 @@
+//go:build unix
+
+package store
+
+import (
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/massalabs/station/pkg/logger"
+)
+
+const databasePattern = "cert*.db"
+
+// NSSDatabases is a struct that represents the known NSS databases on the system.
+type NSSDatabases struct {
+	Paths []string
+}
+
+// NewNssDatabases returns a new NSSDatabases instance.
+func NewNssDatabases() (*NSSDatabases, error) {
+	theoricPath, err := NSSDBPaths()
+	if err != nil {
+		return nil, err
+	}
+
+	return &NSSDatabases{
+		Paths: filterExistingPath(theoricPath),
+	}, nil
+}
+
+// executeOnPaths executes the given operation on each NSS database path.
+func (n *NSSDatabases) executeOnPaths(operation func(path string) error) error {
+	for _, path := range n.Paths {
+		if err := operation(path); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// Add adds the certificate to the NSS databases.
+func (n *NSSDatabases) Add(certPath string, certificateName string) error {
+	return n.executeOnPaths(func(path string) error {
+		err := runCertutilCommand("-A", "-d", path, "-t", "C,,", "-n", certificateName, "-i", certPath)
+		logger.Logger.Debugf("adding the certificate to the NSS database (%s): %s", path, err)
```

Comment:
Here it's logging `adding the certificate to the NSS database (/Users/nathanmassa/Library/Application Support/Firefox/Profiles/k0e9460f.default-release): %!s(<nil>)`

```suggestion
		if err != nil {
			logger.Logger.Debugf("fail adding the certificate to the NSS database (%s): %s", path, err)
		}
```